numpy/core/fromnumeric.py

```diff
@@ -3243,12 +3243,14 @@ def around(a, decimals=0, out=None):
     See Also
     --------
     ndarray.round : equivalent method
-
+    numpy.round_ : equivalent method
```

Comment:
I don't think we should be linking to `round_` - users should probably prefer `numpy.round`, which is literally the same function with a different (less clunky) name.

Reply:
Both `np.round` and `np.round_` are backwards-compatibility aliases. Neither should be used, and when they are listed as aliases, we should probably note that whenever they are mentioned. They have equivalent status and should probably both be listed here as disrecommended backwards-compatibility aliases.

IIRC, we originally had `numpy.round`, but that shadowed the builtin `round()` when `from numpy import *` was used (which was common at the time). For backwards compatibility, we renamed it to `np.round_` and left `np.round` as an alias but removed it from the `np.__all__` list. Sometime later, we decided that the underscore-appending naming scheme was too clunky for the official preferred versions of such functions and settled on the `a` prefix (`around`, `amin`, `amax`, etc.).